src/main/java/org/springframework/data/elasticsearch/annotations/Document.java

```diff
@@ -54,8 +54,14 @@
 	String indexName();
 
 	/**
-	 * Mapping type name.
+	 * Mapping type name. <br/>
+	 * deprecated as Elasticsearch does not support this anymore
+	 * (https://www.elastic.co/guide/en/elasticsearch/reference/7.3/removal-of-types.html) and will remove it in
```

Comment:
We should use proper links with `@see <a href=…` or `@link <a href=…`.

Reply:
done